### PR TITLE
Tested Customer Service Methods for Functionality Null

### PR DESCRIPTION
**Pull request by parksamuelsoung**
All tests passed! I used mockito, is that okay with yall?? Basic functionality, null. More to come later though!!
Screenshot (83)

### CODE REVIEW

**Review by mdsimms91 — DISMISSED** [review]

**Comment by parksamuelsoung** [comment]
Removed rep test.

**Review by sod2003 — APPROVED** [review]
Approved! Great use of new tooling with Mockito.

File: src/test/java/com/skillstorm/spartanwireless/services/impl/TestCustomerServiceImpl.java [hunk 1]
```diff
@@ -1,56 +1,85 @@
...
+        assertNotNull(customerResponseDto);
     }
 
     @Test
-    public void TestDeleteCustomerById() {
-        // TODO Create test for Service's deleteCustomerById method
-        // Remember to use isArchived functionality
+    public void CustomerService_DeleteCustomer_ReturnVoid() {
+        customerServiceImpl.deleteCustomerById(custId1);
+        verify(customerRepository, times(1)).deleteById(custId1);
```
Comment (sod2003) [inline src/test/java/com/skillstorm/spartanwireless/services/impl/TestCustomerServiceImpl.java]: I missed this last time. We need to verify that it's not deleting, but archiving.

**Review by sod2003 — CHANGES_REQUESTED** [review]
We need to verify through testing that we are not deleting customer records, but actually toggling the isArchived value to a value of True;

File: src/test/java/com/skillstorm/spartanwireless/services/impl/TestCustomerServiceImpl.java (hunk 1 above)
Comment (parksamuelsoung) [reply src/test/java/com/skillstorm/spartanwireless/services/impl/TestCustomerServiceImpl.java]: yeah i need to implement that logic.

**Pull request closed without merging**